roles/nginx/tasks/mdeditor.yml

```diff
@@ -0,0 +1,34 @@
+---
+- name: Fetch the mdeditor archive
+  get_url:
+    url: "{{ mdeditor.url }}"
+    dest: /var/www/georchestra/htdocs/mdeditor.zip
+
+- name: Unzip the archive
+  unarchive:
+    src: /var/www/georchestra/htdocs/mdeditor.zip
+    dest: /var/www/georchestra/htdocs/
+    remote_src: true
```

Comment:
I think you can use the `unarchive` module with a `src` set to the url directly:
https://docs.ansible.com/ansible/latest/collections/ansible/builtin/unarchive_module.html#parameter-src, if you want to merge these 2 steps in one.

Reply:
> Ok, it's not used here or anywhere else in this repo, but technically we .. should ? could ?

as long as we can trust the network / github not being hijacked ... we should be safe, I guess ?

Reply:
lets blindly trust github, what could go wrong ;)